File: basics/2-import.py
# here's the same example, but using Python - much more familiar to me :((
import requests

print("Hello, GO Standard Library!")

response = requests.get("http://jsonplaceholder.typicode.com/posts/1")

if response.status_code == 200:
    print(f"HTTP Response Status: {response.status_code}")

else:
    print(f"Error: {response.status_code} {response.reason}")

File: basics/2-import.go
package main

// we can import multiple modules from the standard library
import (
	foo "fmt"  // we can also assign a custom name to a module
	"net/http"
)

func main() {
	foo.Println("Hello, GO Standard Library!") // now we specify our custom module name instead of a standard name

	response, error := http.Get("https://jsonplaceholder.typicode.com/posts/1")

	if error != nil {
		foo.Println("Error:", error)
		return
	}

	defer response.Body.Close()

	foo.Println("HTTP Response Status:", response.Status)
}
